### PR TITLE
Add model Plant with validations

### PR DESCRIPTION
**Pull request by sandrafojcik**
Stworzenie modelu Plant dla roślinek i utworzenie dla nich walidacji dotyczących częstotliwości podlewania oraz ich stanu.

### CODE REVIEW

File: db/migrate/20190707093458_create_plants.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreatePlants < ActiveRecord::Migration[5.2]
+  def change
+    create_table :plants do |t|
+      t.string :name
+      t.integer :irrigation_frequency
```
Comment (jerryskye) [inline db/migrate/20190707093458_create_plants.rb]: This will be measured in days, right? :smile:

File: app/models/plant.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class Plant < ApplicationRecord
+  enum health_status: [:dead, :wither, :bloom, :normal]
+
+  validates :health_status, inclusion: { in: health_statuses.keys }
```
Comment (jerryskye) [inline app/models/plant.rb]: Did you test this validation? Are you 100% sure it works? 🙂

File: db/migrate/20190707093458_create_plants.rb (hunk 1 above)
Comment (sandrafojcik) [reply db/migrate/20190707093458_create_plants.rb]: Yes 👍🏻

Comment (jerryskye) [reply db/migrate/20190707093458_create_plants.rb]: Maybe it's a good idea to rename this column to `:irrigation_frequency_in_days`? What do you think @sleaz0id? :relaxed:
```suggestion
      t.integer :irrigation_frequency_in_days
```

File: app/models/plant.rb (hunk 2 above)
Comment (sandrafojcik) [reply app/models/plant.rb]: Yup

File: db/migrate/20190707093458_create_plants.rb (hunk 1 above)
Comment (sleaz0id) [reply db/migrate/20190707093458_create_plants.rb]: Absolutely, great idea 💪.

File: app/models/plant.rb [hunk 3]
```diff
@@ -2,7 +2,7 @@ class Plant < ApplicationRecord
   enum health_status: [:dead, :wither, :bloom, :normal]
 
   validates :health_status, inclusion: { in: health_statuses.keys }
-  validates :name, :irrigation_frequency, :health_status, presence: true
+  validates :name, :irrigation_frequency_in_days, :health_status, presence: true
```
Comment (sandrafojcik) [inline app/models/plant.rb]: Changes done @jerryskye 🐤

**Pull request merged**